app/src/main/res/layout/item_message.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/ivMatchPhoto"
+        android:layout_width="@dimen/image_width_height"
+        android:layout_height="@dimen/image_width_height"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:srcCompat="@tools:sample/avatars" />
+
+    <TextView
+        android:id="@+id/tvMatchName"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/name_margin_left"
+        android:textStyle="bold"
+        app:layout_constraintStart_toEndOf="@+id/ivMatchPhoto"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="iLoveComics29" />
+
+    <TextView
+        android:id="@+id/tvMessageBody"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/body_margin_left"
+        android:layout_marginTop="@dimen/body_padding_vertical"
+        app:layout_constraintStart_toEndOf="@+id/ivMatchPhoto"
+        app:layout_constraintTop_toBottomOf="@+id/tvMatchName"
+        tools:text="This is my text" />
```

Comment:
is this needed?

Reply:
This is just used for me to see a default value in the design mode so I can better understand what it will look like in practice